.travis.yml

```diff
@@ -1,20 +1,6 @@
-language: node_js
+version: ~> 1.0
 
-node_js:
-  - "12"
-  - "node"
-
-sudo: false
-
-install:
-  - "npm install"
-  - "npm install @hapi/hapi@$HAPI_VERSION"
-
-env:
```

Comment:
do we need to include the env variables?

Reply:
This will depend on how we decide to handle the multiple installation step. There is an ongoing PR in hapijs/ci-config-travis.